src/main/java/com/denizenscript/depenizen/bukkit/events/jobs/JobsJobsExpGainScriptEvent.java

```diff
@@ -0,0 +1,125 @@
+package com.denizenscript.depenizen.bukkit.events.jobs;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.events.ScriptEvent;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import com.denizenscript.depenizen.bukkit.objects.jobs.JobsJobTag;
+import com.gamingmesh.jobs.Jobs;
+import com.gamingmesh.jobs.api.JobsExpGainEvent;
+import com.gamingmesh.jobs.container.ActionType;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class JobsJobsExpGainScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // jobs player earns exp for <'job'> (action <'action'>)
+    //
+    // @Cancellable true
+    //
+    // @Triggers when a player performs an action that would cause them to earn Jobs exp for a certain job
+    //
+    // @Context
+    // <context.job> Returns the job that the player is gaining exp for
+    // <context.experience> Returns the amount of exp the player will earn
+    // <context.action> Returns the name of the action being paid for, which can be: Break, StripLogs, TNTBreak, Place, Kill, MMKill, Fish, Craft, VTrade, Smelt, Brew, Enchant, Repair, Breed, Tame, Dye, Shear, Milk, Explore, Eat, custom-kill, Collect, Bake.
```

Comment:
If there's a javadoc link or similar that contains this enum list, that would be preferred.

Reply:
All I've got is a source file with their definitions if that will satisfy the criteria: [Jobs Github](https://github.com/Zrips/Jobs/blob/master/src/main/java/com/gamingmesh/jobs/container/ActionType.java)